packages/react-composites/tests/README.md

```diff
@@ -25,11 +25,16 @@ Both hermetic and live tests use [playwright](https://playwright.dev/) test fram
 * [../playwright.config.live.ts](../playwright.config.live.ts) for live tests
 
 ## Setup
-
+* Create `.env` file under `packages/react-composites/tests/browser` and in this file, make sure you have defined:
+     *  CONNECTION_STRING
+     *  TEST_DIR
+     *  SNAPSHOT_DIR
+     *  PLAYWRIGHT_OUTPUT_DIR
 * Build all dependencies. Inside `packages/react-composite`, run
   ```sh
   rush build -t .
   ```
+  (if there were some build failures, run `rush update` or `rush update -p`)
```

Comment:
Shouldn't  we just always encourage developers to run a `rush update` at first any ways?

Reply:
thanks updated